### DISCUSSION TITLE
想问一下yihui的最新的日志都更新在哪里呀

### DISCUSSION THREAD

**Discussion by HuizhiXu** [rbind/yihui#1551] [General]
想问一下有人知道yihui最新的日志还会更新吗？之前说每年7月更新，等了好久好久，博客依然是到22年7月。还会发日志吗？

**Comment by yihui** [comment]
多谢关注。今年陆续有些波折需要处理，写日志的时间不太够用，所以写的不多，都存在本地、没有发布。这一年已经马上到头了，今年肯定是不会更新了，明年啥时候更新还不确定；最终会发布，只是目前不好预测时间。不过这些都不重要，真正重要的是你自己要坚持写啊。其实我偶尔还会溜到你的网站上“检查作业”，像最近的那篇贝叶斯就写得很好啊（数学公式的 JS 库是忘了配置还是不知道怎么弄）。
